content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,82 @@ weight: 20
-
-## Standard API terminology
-
-Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, like a pod or namespace. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects will have a unique name to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
+Kubernetes supports efficient change notifications on resources via *watches*.
+Kubernetes also provides consistent list operations so that API clients can
+effectively cache, track, and synchronize the state of resources.
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+You can view the [API reference](/docs/reference/kubernetes-api/) online,
+or read on to learn about the API in general.
 
-* A **resource type** is the name used in the URL (`pods`, `namespaces`, `services`)
-* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**
-* A list of instances of a resource type is known as a **collection**
-* A single instance of the resource type is called a **resource**
-
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource type will be deleted when its namespace is deleted and access to that resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
+<!-- body -->
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages common RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation (their object schema) which is called a *kind*
+* A list of instances of a resource is known as a *collection*
+* A single instance of a resource type is called a *resource*, and also usually represents an *object*
+* For some resource types, the API includes one or more *sub-resources*, which are represented as URI paths below the resource
+
+Most Kubernetes API resource types are
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects):
+they represent a concrete instance of a concept on the cluster, like a
+pod or namespace. A smaller number of API resource types are *virtual* in
```

Comment:
nit: Are there two types of resource:
- object
- virtual

Reply:
That's a question for WG API Expression, I think (I don't know the answer).